packages/ledger-amino/src/launchpadledger.ts

```diff
@@ -166,6 +167,16 @@ export class LaunchpadLedger {
     await this.verifyCosmosAppIsOpen();
   }
 
+  public async verifyAddress(hdPath?: HdPath): Promise<AddressAndPublicKeyResponse> {
+    await this.verifyDeviceIsReady();
+
+    const hdPathToUse = hdPath || this.hdPaths[0];
+    // ledger-cosmos-js hardens the first three indices
+    const response = await this.app.showAddressAndPubKey(unharden(hdPathToUse), this.prefix);
+    this.handleLedgerErrors(response);
+    return response as AddressAndPublicKeyResponse;
```

Comment:
We trun all errors to exceptions in the first line. So we know there is no error anymore in the result and we can use `AddressAndPubkey` instead of `AddressAndPublicKeyResponse`.